modules/cnf-scheduling-numa-aware-workloads.adoc

```diff
@@ -30,7 +30,7 @@ $ oc get numaresourcesschedulers.nodetopology.openshift.io numaresourcesschedule
 .Example output
 [source,terminal]
 ----
```

Comment:
a bit earlier in this section, perhaps in the entry, let's clearly add that "...Now that topology-aware scheduling operator is installed, numaresourcesoperator and numaresourcesschedulers CRs are applied and your cluster has a matching performanceprofile or kubeletconfig, (next sentence is taken from the current documentation for this section) You can schedule workloads with the NUMA-aware scheduler using Deployment CRs that specify the minimum required resources to process the workload."